packages/editor/src/components/block-compare/block-view.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+
+const BlockView = ( { title, rawContent, renderedContent, action, actionText, className } ) => {
+	return (
+		<div className={ className }>
+			<div className="editor-block-compare__content">
+				<h1 className="components-modal__header-heading">{ title }</h1>
```

Comment:
This is not the modal component, so should not be stealing / inheriting styles. Components should be insulated from one another to avoid maintenance issues with cascading consequences of future changes.